source/accounts/accounts_and_projects.rst

```diff
@@ -25,20 +25,27 @@ through them.
 
 .. image:: /images/access1.png
 
-Once you have a NOAA.gov email address, you can request an RDHPCS
-account through the Account Information Managment system, AIM. Visit
-the `Account Information Management`_ website and request
-access to the RDHPCS project.  Log into AIM using your NOAA email
-credentials, review your profile for accuracy, and request the RDHPCS
-project.
+Once you have a NOAA.gov email address, you can:
+
+1. Request an RDHPCS account through the Account Information Managment
+system, **AIM**.
+
+2. Visit the `Account Information Management`_ website and request
```

Comment:
Should steps 2 and 3 be combined?   (They both end with "request the RDHPCS project...")

Reply:
Yes they should. Thanks.